test/Mustache/Test/FiveThree/Functional/StrictVariablesTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of Mustache.php.
+ *
+ * (c) 2017 Enalean
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+/**
+ * @group lambdas
+ * @group functional
+ */
+class Mustache_Test_FiveThree_Functional_StrictVariablesTest extends PHPUnit_Framework_TestCase
```

Comment:
this doesn't need to be a `FiveThree` test because it doesn't use Closures. move it to `Mustache/Test/Functional`?